main.js

```diff
@@ -0,0 +1,21 @@
+const listedLinks = document.querySelectorAll('li')
+for (let listedLink of listedLinks) {
+    listedLinks[0].remove()
+}
+
+const reads = document.querySelectorAll('span')
+for (let read of reads) {
+    read.remove()
+}
+
+const titles = document.querySelectorAll('h3')
+for (let title of titles) {
+    title.remove()
+}
+
+const clientPosts = document.querySelectorAll('article')
```

Comment:
since there are multiple articles and the instructions specified which one should be removed you argument for query selector needs to be more specific.